LibrarySystemBackend/LibrarySystem/LibrarySystem/Services/MemberService.cs

```diff
@@ -23,10 +23,12 @@ public async Task<bool> AddMemberAsync(MemberViewModel memberViewModel)
             {
                 var member = new Member
                 {
-                    MemberID = memberViewModel.MemberID,
+                    Id = memberViewModel.MemberID,
```

Comment:
Do not need to add Id to Add member method (because it is an auto-generated value)